src/workspace-data/provenance/workspace-data-provenance-utils.js

```diff
@@ -180,13 +186,26 @@ export const useFileProvenance = (workspace, fileUrl) => {
     const loadFileProvenance = async () => {
       setError(null);
       setLoading(true);
+      const protocol = getFileProtocol(fileUrl); // for mixpanel event
+      let success = true;
+      let provenanceResultType = 'n/a';
```

Comment:
the `success` and `provenanceResultType` variables jump through a few hoops to prevent `useEffect` from having additional dependencies. I tried reading directly from the `error` and `fileProvenance` state values, but those caused additional dependencies.